sdk/python/endpoints/batch/imagenet-classifier-batch.ipynb

```diff
@@ -0,0 +1,2832 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "# Using batch deployments for image file processing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "Importing the required libraries. This notebook requires:\n",
+    "\n",
+    "- `azure-ai-ml`\n",
+    "- `mlflow`\n",
+    "- `azureml-mlflow`\n",
+    "- `numpy`\n",
+    "- `pandas`\n",
+    "- `pillow`\n",
+    "- `tensorflow`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient, Input\n",
+    "from azure.ai.ml.entities import (\n",
+    "    BatchEndpoint,\n",
+    "    BatchDeployment,\n",
+    "    Model,\n",
+    "    AmlCompute,\n",
+    "    Data,\n",
+    "    BatchRetrySettings,\n",
+    "    CodeConfiguration,\n",
+    "    Environment,\n",
+    ")\n",
+    "from azure.ai.ml.constants import AssetTypes, BatchDeploymentOutputAction\n",
+    "from azure.identity import DefaultAzureCredential"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "## Accessing the Azure Machine Learning workspace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Class RegistryOperations: This is an experimental class, and may change at any time. Please see https://aka.ms/azuremlexperimental for more information.\n"
+     ]
+    }
+   ],
+   "source": [
+    "subscription_id = \"<subscription>\"\n",
+    "resource_group = \"<resource-group>\"\n",
+    "workspace = \"<workspace>\"\n",
+    "\n",
+    "ml_client = MLClient(\n",
+    "    DefaultAzureCredential(), subscription_id, resource_group, workspace\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "## About the model\n",
+    "\n",
+    "Let's review how the model is build. The model was built using TensorFlow along with the RestNet architecture ([Identity Mappings in Deep Residual Networks](https://arxiv.org/abs/1603.05027)). This model has the following constrains that are important to keep in mind for deployment:\n",
```

Comment:
nit: how the model is **built**

Reply:
I think some of these cells are copy pasted from the other noteobook I just reviewed, hence the same comments applies to this notebook :)

Reply:
Done!